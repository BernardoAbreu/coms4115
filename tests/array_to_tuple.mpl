tuple Tup {a, b:real, c:integer}

t = Tup;

a = ["hugo","3.65","22"];

t = a;

print(t);