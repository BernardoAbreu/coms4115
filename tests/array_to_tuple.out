(hugo, 3.65, 22)
